src/hydrogen/transfer.ts

```diff
@@ -0,0 +1,59 @@
+export interface QueryGetTransferPayloadsRequest {
```

Comment:
Would be more user-friendly if you make all fields optional and you added defaults on behalf of the user